highlighting/definition.py

```diff
@@ -60,7 +60,8 @@
 	"input",
 	"or",
 	"while",
-	"length"
+	"length",
+	"comma"
```

Comment:
Can't comment on line 71 and 77 but it is unclear to me shy we have a difference with and without groups, can you add that in a comment?

Reply:
I will add comments on this function to improve the readability.

To make a long story short, there is an option `withoutGroup` which allows you to specify if you want the result to be a group (with parenthesis) or not.

Most of the time, since we handle keywords separately, we want a keyword (and its translations) to form a group (hence the default value)

But in very rare cases, we want to make the groups manually (like for colors), and that's what the option is for.